src/fiatconnect/FiatDetailsScreen.test.tsx

```diff
@@ -137,7 +137,7 @@ describe('FiatDetailsScreen', () => {
 
     const { getByTestId, getByText } = render(<Provider store={store}>{headerTitle}</Provider>)
 
-    expect(getByText('fiatDetailsScreen.header')).toBeTruthy()
+    expect(getByText('fiatDetailsScreen.headerBankAccount')).toBeTruthy()
```

Comment:
add a test checking for mobile money header. Could just use it.each on this test with two different quotes (one for BankAccount and one for MobileMoney).